libs/auth/config/openssl.vx.cnf

```diff
@@ -0,0 +1,12 @@
+# In production, this include is replaced with `.include /etc/ssl/openssl.default.cnf` to avoid a
+# circular dependency. See src/intermediate-scripts/manage-openssl-config for more details.
+.include /etc/ssl/openssl.cnf
```

Comment:
Overall happy with how this piece turned out! No more copying the default OpenSSL config file into the repo and editing it.

This gives `vxsuite-build-system` the ability to define a default OpenSSL config file that's FIPS-compliant, while still allowing local dev without FIPS-compliant OpenSSL.

I also like that this clearly separates application-specific OpenSSL config file logic (like Vx OIDs) from the rest of the config file logic.